go/ql/lib/semmle/go/dataflow/FunctionInputsAndOutputs.qll

```diff
@@ -299,8 +309,13 @@ private class OutParameter extends FunctionOutput, TOutParameter {
   override DataFlow::Node getExitNode(DataFlow::CallNode c) {
     exists(DataFlow::Node arg |
       arg = getArgument(c, index) and
-      // exclude slices passed to varargs parameters using `...` calls
-      not (c.hasEllipsis() and index = c.getNumArgument() - 1)
+      not (
```

Comment:
Same here, except apparently we want to exclude explicit varargs arrays, so syntactic argument except a `...` expression?